src/lib.rs

```diff
@@ -851,65 +853,83 @@ impl LightningNode {
         Ok(payment_hash)
     }
 
-    /// Get a list of the latest payments
+    /// Get a list of the latest movements
     ///
     /// Parameters:
-    /// * `number_of_completed_payments` - the maximum number of completed payments that will be returned
-    pub fn get_latest_payments(
+    /// * `number_of_completed_movements` - the maximum number of completed movements that will be returned
+    pub fn get_latest_movements(
         &self,
-        number_of_completed_payments: u32,
-    ) -> Result<ListPaymentsResponse> {
-        const LEEWAY_FOR_PENDING_OUTGOING_PAYMENTS: u32 = 10;
+        number_of_completed_movements: u32,
+    ) -> Result<ListMovementsResponse> {
+        const LEEWAY_FOR_PENDING_PAYMENTS: u32 = 10;
         let list_payments_request = ListPaymentsRequest {
-            filters: Some(vec![PaymentTypeFilter::Sent, PaymentTypeFilter::Received]),
+            filters: Some(vec![
+                PaymentTypeFilter::Sent,
+                PaymentTypeFilter::Received,
+                PaymentTypeFilter::ClosedChannel,
+            ]),
             from_timestamp: None,
             to_timestamp: None,
             include_failures: Some(true),
-            limit: Some(number_of_completed_payments + LEEWAY_FOR_PENDING_OUTGOING_PAYMENTS),
+            limit: Some(number_of_completed_movements + LEEWAY_FOR_PENDING_PAYMENTS),
             offset: None,
         };
-        let breez_payments = self
+        let breez_movements = self
             .rt
             .handle()
             .block_on(self.sdk.list_payments(list_payments_request))
             .map_to_runtime_error(RuntimeErrorCode::NodeUnavailable, "Failed to list payments")?
             .into_iter()
-            .map(|p| self.payment_from_breez_payment(p))
-            .filter_map(filter_out_and_log_corrupted_payments)
+            .map(|p| self.movement_from_breez_payment(p))
+            .filter_map(filter_out_and_log_corrupted_movements)
             .collect::<Vec<_>>();
 
         // Query created invoices, filter out ones which are in the breez db.
-        let breez_payment_hashes: HashSet<_> = breez_payments
+        let breez_payment_hashes: HashSet<_> = breez_movements
             .iter()
-            .map(|p| p.invoice_details.payment_hash.as_ref())
+            .filter_map(|m| match m {
+                Movement::Payment { payment } => {
+                    Some(payment.invoice_details.payment_hash.as_ref())
+                }
+                Movement::ChannelClose { .. } => None,
+            })
             .collect();
         let created_invoices = self
             .data_store
             .lock_unwrap()
-            .retrieve_created_invoices(number_of_completed_payments)?;
-        let created_invoices = created_invoices
+            .retrieve_created_invoices(number_of_completed_movements)?;
+        let created_invoice_movements = created_invoices
```

Comment:
Not sure how idiomatic that variable name is. Honestly, I have a hard time figuring out what a `created_invoice_movement` is. What exactly is stored in it? Isn't it the last `number_of_completed_movements` invoices minus the confirmed ones (= payments that come from the Breez API?). So is it `unpaid_inbound_invoices` ? Since you map it into a payment, we could also call it `unpaid_inbound_payments` or `pending_inbound_payments`, but why calling it `...movements`? `payment_from_created_invoice` sounds to me like this can only be payments, never channel closes.